uwblueprint/ccbc
# [Feat] Search and Filter Page Background Image

## Notion ticket link

[Search and Filter BG Image](https://www.notion.so/uwblueprintexecs/16fb020b93f641c8b384f76a1b3c769e?v=aedc8d99cd8b437cb556ece10f3c0e3c&p=130533dc9d8a462b824114f5d761b348&pm=c)



## Implementation description
* added background image 
* background image doesn't show in mobile view



## Steps to test
1. Go to http://localhost:3000/magazine/search_results/ and verify that background image works as expected.
2. In file `mockReviews.ts` you can add / delete reviews and check that the background image size works as expected.
3. Go to mobile view and verify that bg image isn't there.

Demo:
https://drive.google.com/file/d/15bGR_BblDI5F_rnQEc_CjjSMWQNytKyw/view?usp=sharing
<img width="1394" alt="Screen Shot 2022-08-03 at 6 42 32 PM" src="https://user-images.githubusercontent.com/53016529/182724880-6035d618-fcfe-4392-b783-1f9a4bc3f37b.png">



## What should reviewers focus on?

## Checklist
